CryptoPrices/Sources/MyCoin/Sources/MyCoin/ChartSectionView/PriceLineChartSection.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  PriceLineChartSection.swift
+//  MyCoin
+//
+//  Created by Minh Pham on 04/01/2023.
+//
+
+import Charts
+import Styleguide
+import SwiftUI
+
+struct PriceLineChartSection: View {
+
+    @State private var highestPriceOffsetX: CGFloat = 0.0
+    @State private var lowestPriceOffsetX: CGFloat = 0.0
+
+    // TODO: Update to use real data in the integrate task
+    // swiftlint:disable number_separator
+    let testData = [
+        ChartDataEntry(x: 1640649600000, y: 50774.067814743),
+        ChartDataEntry(x: 1640736000000, y: 47725.14804631933),
+        ChartDataEntry(x: 1640822400000, y: 46506.99464767938),
+        ChartDataEntry(x: 1640908800000, y: 47191.86838983951),
+        ChartDataEntry(x: 1640995200000, y: 46319.65108805251),
+        ChartDataEntry(x: 1641081600000, y: 47816.07767640849),
+        ChartDataEntry(x: 1641168000000, y: 47387.212167697246),
+        ChartDataEntry(x: 1641254400000, y: 46531.140860530526),
+        ChartDataEntry(x: 1641340800000, y: 45938.02427172366),
+        ChartDataEntry(x: 1641427200000, y: 43647.147508068054)
+    ]
+
+    var body: some View {
+
+        // TODO: Update to use real data in the integrate task
+        GeometryReader { geometry in
+            VStack(alignment: .center, spacing: 0.0) {
+                Text("50774.067814743")
```

Comment:
Although it currently shows the hardcoded data, it would be great if the data was in the correct format (i.e. `dollarCurrency`) 🙏